lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -542,7 +542,8 @@ function dosomething_campaign_get_campaigns_doing($uid = NULL) {
  */
 function dosomething_campaign_get_campaign_block_vars($nid, $image_size = '740x480', $source = NULL) {
   $node = node_load($nid);
-  $clc = dosomething_helpers_get_current_language_content_code();
```

Comment:
i know this function was being used to fix some things please check comments here
https://github.com/DoSomething/phoenix/blob/44e8094e68ef24dcfe9c4e8875c68960ab52578f/lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module#L607